### PR TITLE
ZK-4808: menu click listener ignored (unexpectedly)

### PR DESCRIPTION
**Pull request by b952426**
ZK-4808: menu click listener ignored (unexpectedly)

### CODE REVIEW

File: zul/src/archive/web/js/zul/menu/less/menu.less [hunk 1]
```diff
@@ -170,6 +170,13 @@
 		right: 0;
 		top: @menuIconPositionTop;
 	}
+	&-icon-area {
+		width: @menuIconSize + 10px;
+		height: 100%;
+		display: inline-block;
+		position: absolute;
+		top: 0;
```
Comment (scribetw) [inline zul/src/archive/web/js/zul/menu/less/menu.less:178]: add `right: 0;`

File: zul/src/archive/web/js/zul/menu/less/menu.less [hunk 2]
```diff
@@ -170,6 +170,13 @@
 		right: 0;
 		top: @menuIconPositionTop;
 	}
+	&-icon-area {
+		width: @menuIconSize + 10px;
```
Comment (scribetw) [inline zul/src/archive/web/js/zul/menu/less/menu.less:174]: use .z-menu-separator right and extract to a variable

**Review by scribetw — CHANGES_REQUESTED** [review]

**Review by scribetw — APPROVED** [review]

**Review by plijyfes — APPROVED** [review]

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**